fix: Make query postgres compatible

surajshetty3416 opened the pull request.

surajshetty3416 committed:
fix: Make query postgres compatible

saurabh6790 commented on frappe/desk/page/activity/activity.py:
```diff
@@ -38,7 +38,7 @@ def get_feed(start, page_length):
 		) X
 		order by X.creation DESC
-		limit %(start)s, %(page_length)s"""
+		OFFSET %(start)s LIMIT %(page_length)s"""
```
will this work with mysql too or its db dependent?

surajshetty3416 replied:
@saurabh6790  Yes, it will work on both... It's Standard SQL

surajshetty3416 committed:
fix: Switch the position of LIMIT and OFFSET

MariaDB needs LIMIT before OFFSET. 
Whereas Postgres accepts LIMIT and OFFSET in any order
